imagetest/utils/test_utils.go

```diff
@@ -105,6 +109,38 @@ func GetMetadataHTTPResponse(path string) (*http.Response, error) {
 	return resp, nil
 }
 
+// PutMetadataGuestAttribute sets the guest attribute in the namespace, and returns an error if this operation fails.
+func PutMetadataGuestAttribute(namespace, attribute string) error {
+	return PutMetadata("guest-attributes/" + namespace + "/" + attribute)
+}
+
+// PutMetadata returns a metadata value for the specified key if it is present, and error if not.
+func PutMetadata(path string) error {
+	resp, err := PutMetadataHTTPResponse(path)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != 200 {
```

Comment:
Can you please handle `resp.StatusCode` in `PutMuetadataHTTPResponse()` instead.

Reply:
done